### DISCUSSION TITLE
user/group text as question marks

### DISCUSSION THREAD

**Discussion by vncsalencar** [lsd-rs/lsd#1042] [Q&A]
![image](https://github.com/lsd-rs/lsd/assets/42007303/1baa8030-003e-42b8-9417-d55d22ca9883)

The user/group is being shown as question marks
I haven't found any issue related to it (or I'm missing on how to search about the issue)

PowerShell 7.4.2
lsd 1.1.2
Windows 11 23H2 OS Build 22631.3527

**Comment by yorah** [comment]
Same here, any idea what could be causing that?
Windows 11 23H2
lsd 1.1.5
git bash (or any other shell)

**Comment by zhangbird050801** [comment]
change the permission in the config.yml to rwx, that's helps me.
permission: rwx
![image](https://github.com/user-attachments/assets/de2d10ab-4da3-4b4d-a326-e5c928158112)
